chainer/link_hooks/weight_standardization.py

```diff
@@ -0,0 +1,67 @@
+import chainer.functions as F
+from chainer import link_hook
+
+
+class WeightStandardization(link_hook.LinkHook):
+    """Weight Standardization link hook implementation.
```

Comment:
Personally, it should be noted that this hook is originally proposed for 2D convolution layers followed by mainly GN and sometimes BN.
Also, I want some disclaimer that this hook is able to handle layers such as N-dimensional convolutional, linear and embedding layers but there is no guarantee that this hook helps training.

Reply:
thank you!